app/javascript/styles/mastodon/admin.scss

```diff
@@ -1681,6 +1681,15 @@ a.sparkline {
   box-sizing: border-box;
   min-height: 100%;
 
+  a {
+    text: &highlight-text-color;
```

Comment:
I think merging this should be delayed until this is addressed upstream.
While I doubt many people using glitch will ever use the vanilla flavour, especially as admins, and this is a very minor thing, I'm not a fan of introducing a known issue.

Reply:
There are other issues being fixed and I don't want to cherry-pick patches, also it doesn't break anything that wasn't broken already in practice.

Reply:
Fair enough